compiler/semdata.nim

```diff
@@ -164,7 +164,7 @@ type
     isAmbiguous*: bool # little hack
     features*: set[Feature]
     inTypeContext*, inConceptDecl*: int
-    unusedImports*: seq[(PSym, TLineInfo)]
+    unusedImports*: seq[PSym]
```

Comment:
If you use the lineinfo from the PSym you rely on the fact that module PSym are duplicated. But I'm not sure we want to commit to this design choice.

Reply:
Done